### PR TITLE
Fix args passed to CaseOnDisk

### PR DESCRIPTION
**Pull request by equinor-ruaj**
Handle changes in uploader: https://github.com/equinor/fmu-sumo-uploader/pull/93

### CODE REVIEW

File: tests/test_functions.py [hunk 1]
```diff
@@ -434,8 +434,25 @@ def test_sim2sumo_with_ert(scratch_files, case_uuid, sumo, monkeypatch):
...
-    results = sumo.get(path).json()
+    path = f"/objects('{case_uuid}')/search"
+    results = sumo.post(
+        path,
+        json={
+            "query": {
+                "bool": {
+                    "must": [{"term": {"fmu.case.uuid.keyword": case_uuid}}],
+                    "must_not": [
+                        {"term": {"class.keyword": "case"}},
```
Comment (rwiker) [inline tests/test_functions.py:445]: Not important, but could be written as

```
{
    "terms": {
        "class.keyword": ["case", "iteration", "realization"]
    }
}
```

File: tests/test_functions.py [hunk 2]
```diff
@@ -434,8 +434,25 @@ def test_sim2sumo_with_ert(scratch_files, case_uuid, sumo, monkeypatch):
...
     expected_exports = 88
-    path = f"/objects('{case_uuid}')/children"
-    results = sumo.get(path).json()
+    path = f"/objects('{case_uuid}')/search"
+    results = sumo.post(
+        path,
+        json={
+            "query": {
+                "bool": {
+                    "must": [{"term": {"fmu.case.uuid.keyword": case_uuid}}],
```
Comment (rwiker) [inline tests/test_functions.py:443]: Don't actually need this, as the this particular endpoint adds a restriction on `case_uuid`.

**Review by rwiker — APPROVED** [review]
LGTM.

File: tests/test_functions.py (hunk 1 above)
Comment (equinor-ruaj) [reply tests/test_functions.py]: Nice 🍠

File: tests/test_functions.py (hunk 2 above)
Comment (equinor-ruaj) [reply tests/test_functions.py]: right.. 🤦

**Pull request merged**